libcxx/include/__exception/exception_ptr.h

```diff
@@ -82,17 +85,42 @@ class _LIBCPP_EXPORTED_FROM_ABI exception_ptr {
   _LIBCPP_HIDE_FROM_ABI exception_ptr() _NOEXCEPT : __ptr_() {}
   _LIBCPP_HIDE_FROM_ABI exception_ptr(nullptr_t) _NOEXCEPT : __ptr_() {}
 
+// These symbols are still exported from the library to prevent ABI breakage.
+#  ifdef _LIBCPP_BUILDING_LIBRARY
```

Comment:
You'll probably need to add availability markups for Apple platforms.

Reply:
I think in this particular case we don't need availability markup.

Afaict, availability markup is needed for newly introduced symbols.
However, I am not introducing a new symbol. I am turning an existing symbol into a no-longer used symbol.

Reply:
You're introducing `__increment_refcount` and `__decrement_refcount`, so we need to provide some other implementation if they're not available. Using the old implementation seems like a rather obvious solution to me.